src/cli/common.js

```diff
@@ -305,20 +305,28 @@ function cli(api){
     function readConfigData(config) {
         var data = readConfigFile(config),
             json,
+            optionName,
+            optionValue,
+            args,
             options = {};
         if (data) {
             if (data.charAt(0) === "{") {
                 try {
                     json = JSON.parse(data);
                     data = "";
-                    for (var optionName in json) {
+                    for (optionName in json) {
                         if (json.hasOwnProperty(optionName)) {
-                            data += "--" + optionName + "=" + json[optionName].join();
+                            optionValue = json[optionName];
+                            if (typeof optionValue === "string") {
+                                optionValue = [optionValue];
+                            }
+                            data += "--" + optionName + "=" + optionValue.join(",");
```

Comment:
Since we want the value to end up as a string in the end, what do you think about checking for `Array.isArray(optionValue)` and then calling `.join(",")` in that case…